Sources/Reactive.swift

```diff
@@ -0,0 +1,26 @@
+/// Describes a class which has been extended with reactive elements.
+///
+/// - note: `ExtendedForReactiveness` is only intended for extensions to types
+///         that are not owned by the module. Non-conforming types may carry
+///         first-party reactive elements.
+public protocol ExtendedForReactiveness: class {}
+
+extension ExtendedForReactiveness {
+	/// A proxy which exposes the reactivity of `self`.
+	public var rac: Reactive<Self> {
```

Comment:
Properties do not have overloading at this moment. Even if they would do (planned), we would probably want to avoid return type ambiguity as much as possible, wouldn't we?